docs/README.md

```diff
@@ -68,9 +68,9 @@ Following sections are Kubernetes specific. If you are Kubernetes user, use foll
 * **Dynamic Provisioning** - uses persistence volume claim (PVC) to request the Kuberenetes to create the EBS volume on behalf of user and consumes the volume from inside container. Storage class's **allowedTopologies** could be used to restrict which AZ the volume should be provisioned in. The topology key should be **topology.ebs.csi.aws.com/zone**.
 * **Mount Option** - mount options could be specified in persistence volume (PV) to define how the volume should be mounted.
 * **NVMe** - consume NVMe EBS volume from EC2 [Nitro instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances).
-* **Block Volume** (beta since 1.14) - consumes the EBS volume as a raw block device for latency sensitive application eg. MySql
-* **Volume Snapshot** (alpha) - creating volume snapshots and restore volume from snapshot.
-* **Volume Resizing** (alpha) - expand the volume size.
+* **Block Volume** ([beta since 1.14](https://kubernetes-csi.github.io/docs/raw-block.html)) - consumes the EBS volume as a raw block device for latency sensitive application eg. MySql
```

Comment:
Seems like this is GA since 1.18 (the linked page must be outdated as well):

https://github.com/kubernetes/kubernetes/blob/43fbe17dc682b703b0b663935a797244378f0aaf/pkg/features/kube_features.go#L283-L287

Reply:
Nice catch, updated the feature state here and proposed https://github.com/kubernetes-csi/docs/pull/350.

Reply:
Is the AWS implementation of block volume considered GA? There is at least one known issue is hit very often on a pod restart on the same node. https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/439

Reply:
I don't think there's any feature considered GA in the EBS CSI driver currently, as the dirver itself ([should be beta still](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/milestones?state=closed).

Note that the graduation level of these features are related to k8s, not the driver.

Reply:
It might be good to be more explicit about what the feature status is at the driver level, and what it is at the k8s level. For example, nvme listed here is not a k8s feature.

Reply:
Since the updated doc is under kubernetes section, it makes sense to update it this way. But yeah we need to be explicit about driver level feature support (which is missing from README). We need to prioritize the fix of $439

Reply:
> It might be good to be more explicit about what the feature status is at the driver level, and what it is at the k8s level. For example, nvme listed here is not a k8s feature.

Good point, @msau42 . I now clarified that the feature status in the description corresponds to the CSI feature status.